tests/case/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -0,0 +1,457 @@
+
+interface iTemplate {
+}
+
+abstract class AbstractClass
+{
+
+  abstract protected function getValue();
+  abstract protected function prefixValue($prefix);
+
+  public function printOut()
+  {
+    print $this->getValue();
+  }
+}
+
+trait ezcReflectionReturnInfo {
+  public function getReturnType()
+  {
+    // 1
+  }
+  public function getReturnDescription()
+  {
+    // 2
+  }
+}
+
+final class MyClass extends Bar implements iTemplate
+{
+
+  use A, B {
+    B::smallTalk insteadof A;
+    A::bigTalk insteadof B;
+    B::bigTalk as talk;
+  }
+
+  const CONSTANT = 'CONSTANT';
+
+  public $public = 'Public';
+
+  protected $protected = 'Protected';
+
+  private $private = 'Private';
+
+  public $foo = 'Bar';
+
+  public function printHello()
+  {
+    echo $this->public;
+
+    $myclass = new MyClass();
+
+    $true = true;
+    $false = false;
+    $null = null;
+    $other_null = null;
+    $null_string = "NULL";
+
+    if ($expr1) {
+      // IF body
+    } elseif ($expr2) {
+      // ELSEIF body
+    } elseif ($expr3) {
+      // ELSE IF body
+    } else {
+      // Else body
+    }
+
+    switch ($expr) {
+      case 0:
+        echo 'First case, with a break';
+        break;
+      case 1:
+      case 4:
+        echo 'Third case, return instead of break';
+        return;
+      default:
+        echo 'Default case';
+        break;
+    }
+
+    while (list($key, $value) = foo($arr)) {
+      if (!($key % 2)) {
+        continue;
+      }
+
+      do_something_odd($value);
+    }
+
+    do {
+      // Structure body
+    } while ($expr);
+
+    for ($i = 0; $i < 10; $i++) {
+      // For body
+    }
+
+    foreach ($iterable as $key => $value) {
+      // Foreach body
+    }
+
+    try {
+      // Try body
+    } catch (FirstExceptionType $e) {
+      // Catch body
+    } finally {
+      restore_error_handler();
+    }
+
+    $noArgs_longVars = function () use (
+      $longVar1,
+      $longerVar2,
+      $muchLongerVar3
+    ) {
+      // Body
+    };
+
+    $obj2 = clone $obj;
+
+    if (empty($var)) {
+      echo '$var или 0, или пусто, или вообще не определена';
+    }
+
+    if (isset($var)) {
+      echo '$var определена, даже если она пустая';
+    }
+
+    __LINE__;
+    __FILE__;
+    __DIR__;
+    __FUNCTION__;
+    __CLASS__;
+    __TRAIT__;
+    __METHOD__;
+    __NAMESPACE__;
+
+    $c = (false and foo());
+    $d = (true or foo());
+    $c = (false xor foo());
+
+    $array = array(1, 1, 1);
+
+    if ($step <= 0) {
+      throw new LogicException('Logic');
+    }
+
+    for ($i = $start; $i <= $limit; $i += $step) {
+      yield $i;
+    }
+
+    global $foo;
+    unset($foo);
+
+    var_dump($a instanceof MyClass);
+
+    exit();
+    exit();
+    eval('test');
+  }
+
+  public function MyPublic()
+  {
+    echo self::CONSTANT;
+  }
+
+  protected function MyProtected()
+  {
+    require ('somefile.php');
+    require_once ('somefile.php');
+    include ('somefile.php');
+    include_once ('somefile.php');
```

Comment:
`require ('somefile.php');` => `require 'somefile.php';` 

All popular framework use this form
https://github.com/laravel/framework/search?l=PHP&q=require&type=&utf8=%E2%9C%93
https://github.com/symfony/symfony/search?utf8=%E2%9C%93&q=require&type=
https://github.com/yiisoft/yii2/search?l=PHP&q=require&type=&utf8=%E2%9C%93

Reply:
Let's do this in other PR better :smile: